test/fixtures/v2/documentRopstenInvalidProofSignature.ts

```diff
@@ -0,0 +1,51 @@
+import { v2, WrappedDocument } from "@govtechsg/open-attestation";
```

Comment:
can you add a comment to explain why the signature not good or add a more obvious invalid signature ?